Gems/UiBasics/CMakeLists.txt

```diff
@@ -7,7 +7,5 @@
 #
 
 # This will export its "SourcePaths" to the generated "cmake_dependencies.<project>.assetbuilder.setreg"
-if(PAL_TRAIT_BUILD_HOST_TOOLS)
-    ly_create_alias(NAME UiBasics.Builders NAMESPACE Gem)
-    ly_create_alias(NAME UiBasics.Tools NAMESPACE Gem)
-endif()
+ly_create_alias(NAME UiBasics.Builders NAMESPACE Gem)
+ly_create_alias(NAME UiBasics.Tools NAMESPACE Gem)
```

Comment:
Builders and Tools are normally gated by `PAL_TRAIT_BUILD_HOST_TOOLS`, why was this removed?